templates/includes/sidebar.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
```

Comment:
I am surprised that it is working becouse you create a loop. In sidebar you extend base and inside base you include sidebar